.github/workflows/ci.yml

```diff
@@ -324,3 +324,40 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - run: echo "Only doc changes"
+
+  telemetry-crwa:
+    needs: check
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        node-version: [16, 18]
+      fail-fast: true
+    name: 🔭 Telemetry check - CRWA / ${{ matrix.os }} / node ${{ matrix.node-version }} latest
+    runs-on: ${{ matrix.os }}
+    env:
+      REDWOOD_REDIRECT_TELEMETRY: "http://127.0.0.1:48619" # Random port
+    steps:
+      - uses: actions/checkout@v3
+      - name: 🧶 Set up job
+        uses: ./.github/actions/set-up-job
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: ⚒️ Build the framework packages
+        run: yarn build
+        working-directory: .
```

Comment:
Nit here: I'm going to change this to be same as all the other calls to `yarn build`

Reply:
Done in https://github.com/redwoodjs/redwood/pull/7623/commits/a67f9ea56d4a6f5e93dad598ea0fba5fabbbf14a